src/node_options.h

```diff
@@ -121,8 +121,8 @@ class EnvironmentOptions : public Options {
   std::string experimental_policy_integrity;
   bool has_policy_integrity_string = false;
   bool experimental_permission = false;
-  std::string allow_fs_read;
-  std::string allow_fs_write;
+  std::vector<std::string> allow_fs_read;
```

Comment:
```suggestion
  std::unordored_map<std::string> allow_fs_read;
```

can we use this instead?

Reply:
Not sure why that would be better. Maybe I'm missing something but isn't unordered map a key-value store?

Reply:
Never mind, I read it differently.